IconDiffBot/Core/Application.cs

```diff
@@ -57,12 +59,15 @@ public void ConfigureServices(IServiceCollection services)
 					builder.UseSqlServerStorage(dbConfig.ConnectionString, new SqlServerStorageOptions { PrepareSchemaIfNecessary = true });
 			});
 
+			services.AddHangfireServer();
+
 			services.Configure<IISOptions>((options) => options.ForwardClientCertificate = false);
-			services.AddMvc();
 			services.AddOptions();
 			services.AddLocalization();
+			services.AddControllersWithViews();
+			services.AddRazorPages();
 
-			services.AddDbContext<DatabaseContext>();
+			services.AddDbContext<DatabaseContext>(ServiceLifetime.Transient);
```

Comment:
Weird, that should've been the default option (If it wasn't that alone is a potentially big leak)

Reply:
I had to add this to make the DBContext not complain about being threaded.